packages/react/src/patterns/sub-patterns/Callout/__stories__/Callout.stories.js

```diff
@@ -0,0 +1,72 @@
+import './index.scss';
+import { object, text, withKnobs } from '@storybook/addon-knobs';
+import Callout from '../Callout';
+import LinkList from '../../LinkList/LinkList';
+import React from 'react';
+import readme from '../README.md';
+import { storiesOf } from '@storybook/react';
+
+storiesOf('Patterns (Sub-Patterns)|Callout', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    readme: {
+      sidebar: readme,
+    },
+  })
+  .add('With Link List example', () => {
```

Comment:
looked at the example and LinkList in this looks weird, I'm wondering if we should pick another example? Perhaps something that has a dark theme associated with it? Or maybe just a button?